cmd/ops_agent_windows/run_windows.go

```diff
@@ -123,21 +139,21 @@ func (s *service) checkForStandaloneAgents(unified *confgenerator.UnifiedConfig)
 	return nil
 }
 
-func (s *service) generateConfigs() error {
+func (s *service) generateConfigs() (confgenerator.UnifiedConfig, error) {
```

Comment:
You may wanna touch main_windows.go to add the new diagnostic service thus it's consistent on what we did for the other two subagents on windows